atd-etl/app/process/socrata_queries.py

```diff
@@ -26,7 +26,7 @@
                       { crash_time: { _is_null: false }}
                     ]
                     _or: [
-                        {austin_full_purpose: {_eq: "Y"}},
+                        {in_austin_full_purpose: {_eq: true}},
                         {
                             _and: [
                                 {city_id: {_eq: 22}},
```

Comment:
I really like the new column name! Looking at the lines below this update, we probably need to update this query since `in_austin_full_purpose` now includes `city_id = 22` and lat/lon is not null.

There may be other places where this logic is present in queries and could be a separate issue to sweep through the code for other places where there could be some overlap before this and the other PR for the `city_id` Lambda are merged. _Edit: Just saw the rest of the PR so it looks like you covered most of these already 🙏_

Reply:
ooh thank you for catching that! ill defs update this query and remove the part about `city_id = 22` since that logic is already captured in the `in_austin_full_purpose` column